Calibration/HcalAlCaRecoProducers/src/AlCaIsoTracksProducer.cc

```diff
@@ -1,206 +1,206 @@
+  std::string                theTrackQuality, processName;
+  std::string                l1Filter, l2Filter, l3Filter;
+  double                     a_mipR, a_coneR, a_charIsoR;
+  double                     pTrackMin_, eEcalMax_, eIsolation_;
+  int                        nRun, nAll, nGood;
+  edm::InputTag              labelTriggerEvent_, labelTriggerResults_, labelBS_;
+  edm::InputTag              labelGenTrack_, labelRecVtx_,  labelHltGT_;
+  edm::InputTag              labelEB_, labelEE_, labelHBHE_;
+  const MagneticField       *bField;
+  const CaloGeometry        *geo;
+  double                     ptL1, etaL1, phiL1;
+
+  edm::EDGetTokenT<trigger::TriggerFilterObjectWithRefs>  tok_hltGT_;
+  edm::EDGetTokenT<trigger::TriggerEvent>                 tok_trigEvt_;
+  edm::EDGetTokenT<edm::TriggerResults>                   tok_trigRes_;
+  edm::EDGetTokenT<reco::TrackCollection>                 tok_genTrack_;
+  edm::EDGetTokenT<reco::VertexCollection>                tok_recVtx_;
+  edm::EDGetTokenT<reco::BeamSpot>                        tok_bs_;
+  edm::EDGetTokenT<EcalRecHitCollection>                  tok_EB_;
+  edm::EDGetTokenT<EcalRecHitCollection>                  tok_EE_;
+  edm::EDGetTokenT<HBHERecHitCollection>                  tok_hbhe_;
+};
+
+
+AlCaIsoTracksProducer::AlCaIsoTracksProducer(const edm::ParameterSet& iConfig) :
+  nRun(0), nAll(0), nGood(0) {
+  //Get the run parameters
+  trigNames                           = iConfig.getParameter<std::vector<std::string> >("Triggers");
+  theTrackQuality                     = iConfig.getParameter<std::string>("TrackQuality");
+  processName                         = iConfig.getParameter<std::string>("ProcessName");
+  l1Filter                            = iConfig.getParameter<std::string>("L1Filter");
+  l2Filter                            = iConfig.getParameter<std::string>("L2Filter");
+  l3Filter                            = iConfig.getParameter<std::string>("L3Filter");
+  selectionParameters.minPt           = iConfig.getParameter<double>("MinTrackPt");
+  selectionParameters.minQuality      = reco::TrackBase::qualityByName(theTrackQuality);
+  selectionParameters.maxDxyPV        = iConfig.getParameter<double>("MaxDxyPV");
+  selectionParameters.maxDzPV         = iConfig.getParameter<double>("MaxDzPV");
+  selectionParameters.maxChi2         = iConfig.getParameter<double>("MaxChi2");
+  selectionParameters.maxDpOverP      = iConfig.getParameter<double>("MaxDpOverP");
+  selectionParameters.minOuterHit     = iConfig.getParameter<int>("MinOuterHit");
+  selectionParameters.minLayerCrossed = iConfig.getParameter<int>("MinLayerCrossed");
+  selectionParameters.maxInMiss       = iConfig.getParameter<int>("MaxInMiss");
+  selectionParameters.maxOutMiss      = iConfig.getParameter<int>("MaxOutMiss");
+  a_coneR                             = iConfig.getParameter<double>("ConeRadius");
+  a_charIsoR                          = a_coneR + 28.9;
```

Comment:
As already noticed by @cvuosalo please do not use magic numbers: what does `28.9` means?